### PR TITLE
[IT] Add support for HassRespond

### PR DESCRIPTION
**Pull request by Sartoric**
Added support for HassRespond for Italian language

### CODE REVIEW

**Comment by auanasgheps** [comment]
Hi, i added some comments, can you please clarify?

**Comment by Sartoric** [comment]
> Hi, i added some comments, can you please clarify?

Hi @auanasgheps, I can't see any comments in Files Changed/Conversation 
Am I missing something ?

**Review by auanasgheps — COMMENTED** [review]
Please check my comments

**Comment by auanasgheps** [comment]
My fault, the review was not submitted, you should see them now

File: responses/it/HassRespond.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: it
+responses:
+  intents:
+    HassRespond:
+      hello: "Ciao da Home Assistant."
+      listening: "No, registro solamente quando usi la wake word."
+      data: "I tuoi dati sono inviati al tuo server Home Assistant."
+      commands: "Per conoscere cosa puoi chiedere, visita il sito home trattino assistant punto IO slash voice trattino basso control."
```
[the remark this answers is not in the document]
Comment (Sartoric) [reply responses/it/HassRespond.yaml:8]: sembra strano anche a me francamente, ma ho fatto riferimento "all'originale" inglese
` visit H.A. dot I.O. slash voice.`

che tra l'altro non funziona più :)

File: tests/it/homeassistant_HassRespond.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    intent:
+      name: HassRespond
+    response: "I tuoi dati sono inviati al tuo server Home Assistant."
+
+  - sentences:
+      - "cosa posso chiederti?"
+      - "cosa posso domandare?"
+    intent:
+      name: HassRespond
+    response: "Per conoscere cosa puoi chiedere, visita il sito home trattino assistant punto IO slash voice trattino basso control."
```
[the remark this answers is not in the document]
Comment (Sartoric) [reply tests/it/homeassistant_HassRespond.yaml:27]: vedi precedente

**Comment by Sartoric** [comment]
No worries.
I thought it was my fault :)

File: responses/it/HassRespond.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (auanasgheps) [reply responses/it/HassRespond.yaml:8]: Perfetto, allora ce lo teniamo così! Grazie

**Comment by auanasgheps** [comment]
Grazie mille!

**Pull request merged**